vtysh/subdir.am

```diff
@@ -37,4 +37,5 @@ $(vtysh_vtysh_OBJECTS): vtysh/vtysh_daemons.h
 
 CLEANFILES += vtysh/vtysh_daemons.h
 vtysh/vtysh_daemons.h:
+	@mkdir -p vtysh
```

Comment:
`mkdir -p` never fails actually, `@` is not needed, IMO?

Reply:
Done. Thanks.